### PR TITLE
Allow user types for mysql

### PR DESCRIPTION
**Pull request by eahrend**
Fixes https://github.com/terraform-google-modules/terraform-google-sql-db/issues/186

It appears there is a test that is failing unrelated to the changes I made, not sure if I should fix that test or file it as a bug, I've added the mysql type to the module. 

Test output:
```
-----> Destroying <mysql-public-local>...
$$$$$$ Verifying the Terraform client version is in the supported interval of >= 0.11.4, < 0.14.0...
$$$$$$ Reading the Terraform client version...
       Terraform v0.13.5
       + provider registry.terraform.io/hashicorp/google v3.78.0
       + provider registry.terraform.io/hashicorp/google-beta v3.78.0
       + provider registry.terraform.io/hashicorp/null v3.1.0
       + provider registry.terraform.io/hashicorp/random v3.1.0
       
       Your version of Terraform is out of date! The latest version
       is 1.0.3. You can update by downloading from https://www.terraform.io/downloads.html
$$$$$$ Finished reading the Terraform client version.
$$$$$$ Finished verifying the Terraform client version.
$$$$$$ Initializing the Terraform working directory...
       Initializing modules...
       
       Initializing the backend...
       
       Initializing provider plugins...
       - Using previously-installed hashicorp/random v3.1.0
       - Using previously-installed hashicorp/null v3.1.0
       - Using previously-installed hashicorp/google v3.78.0
       - Using previously-installed hashicorp/google-beta v3.78.0
       
       The following providers do not have any version constraints in configuration,
       so the latest version was installed.
       
       To prevent automatic upgrades to new major versions that may contain breaking
       changes, we recommend adding version constraints in a required_providers block
       in your configuration, with the constraint strings suggested below.
       
       * hashicorp/google-beta: version = "~> 3.78.0"
       
       Terraform has been successfully initialized!
$$$$$$ Finished initializing the Terraform working directory.
$$$$$$ Selecting the kitchen-terraform-mysql-public-local Terraform workspace...
       
       Workspace "kitchen-terraform-mysql-public-local" doesn't exist.
       
       You can create this workspace with the "new" subcommand.
$$$$$$ Creating the kitchen-terraform-mysql-public-local Terraform workspace...
       Created and switched to workspace "kitchen-terraform-mysql-public-local"!
       
       You're now on a new, empty workspace. Workspaces isolate their state,
       so if you run "terraform plan" Terraform will not see any existing state
       for this configuration.
$$$$$$ Finished creating the kitchen-terraform-mysql-public-local Terraform workspace.
$$$$$$ Destroying the Terraform-managed infrastructure...
       
       Destroy complete! Resources: 0 destroyed.
$$$$$$ Finished destroying the Terraform-managed infrastructure.
$$$$$$ Selecting the default Terraform workspace...
       Switched to workspace "default".
$$$$$$ Finished selecting the default Terraform workspace.
$$$$$$ Deleting the kitchen-terraform-mysql-public-local Terraform workspace...
       Deleted workspace "kitchen-terraform-mysql-public-local"!
$$$$$$ Finished deleting the kitchen-terraform-mysql-public-local Terraform workspace.
       Finished destroying <mysql-public-local> (0m7.13s).
-----> Destroying <postgresql-public-local>...
$$$$$$ Verifying the Terraform client version is in the supported interval of >= 0.11.4, < 0.14.0...
$$$$$$ Reading the Terraform client version...
       Terraform v0.13.5
       + provider registry.terraform.io/hashicorp/google v3.78.0
       + provider registry.terraform.io/hashicorp/google-beta v3.78.0
       + provider registry.terraform.io/hashicorp/null v3.1.0
       + provider registry.terraform.io/hashicorp/random v3.1.0
       
       Your version of Terraform is out of date! The latest version
       is 1.0.3. You can update by downloading from https://www.terraform.io/downloads.html
$$$$$$ Finished reading the Terraform client version.
$$$$$$ Finished verifying the Terraform client version.
$$$$$$ Initializing the Terraform working directory...
       Initializing modules...
       
       Initializing the backend...
       
       Initializing provider plugins...
       - Using previously-installed hashicorp/google-beta v3.78.0
       - Using previously-installed hashicorp/random v3.1.0
       - Using previously-installed hashicorp/google v3.78.0
       - Using previously-installed hashicorp/null v3.1.0
       
       The following providers do not have any version constraints in configuration,
       so the latest version was installed.
       
       To prevent automatic upgrades to new major versions that may contain breaking
       changes, we recommend adding version constraints in a required_providers block
       in your configuration, with the constraint strings suggested below.
       
       * hashicorp/google-beta: version = "~> 3.78.0"
       
       Terraform has been successfully initialized!
$$$$$$ Finished initializing the Terraform working directory.
$$$$$$ Selecting the kitchen-terraform-postgresql-public-local Terraform workspace...
       
       Workspace "kitchen-terraform-postgresql-public-local" doesn't exist.
       
       You can create this workspace with the "new" subcommand.
$$$$$$ Creating the kitchen-terraform-postgresql-public-local Terraform workspace...
       Created and switched to workspace "kitchen-terraform-postgresql-public-local"!
       
       You're now on a new, empty workspace. Workspaces isolate their state,
       so if you run "terraform plan" Terraform will not see any existing state
       for this configuration.
$$$$$$ Finished creating the kitchen-terraform-postgresql-public-local Terraform workspace.
$$$$$$ Destroying the Terraform-managed infrastructure...
       
       Destroy complete! Resources: 0 destroyed.
$$$$$$ Finished destroying the Terraform-managed infrastructure.
$$$$$$ Selecting the default Terraform workspace...
       Switched to workspace "default".
$$$$$$ Finished selecting the default Terraform workspace.
$$$$$$ Deleting the kitchen-terraform-postgresql-public-local Terraform workspace...
       Deleted workspace "kitchen-terraform-postgresql-public-local"!
$$$$$$ Finished deleting the kitchen-terraform-postgresql-public-local Terraform workspace.
       Finished destroying <postgresql-public-local> (0m6.93s).
-----> Destroying <postgresql-public-iam-local>...
$$$$$$ Verifying the Terraform client version is in the supported interval of >= 0.11.4, < 0.14.0...
$$$$$$ Reading the Terraform client version...
       Terraform v0.13.5
       + provider registry.terraform.io/hashicorp/google v3.78.0
       + provider registry.terraform.io/hashicorp/google-beta v3.78.0
       + provider registry.terraform.io/hashicorp/null v3.1.0
       + provider registry.terraform.io/hashicorp/random v3.1.0
       
       Your version of Terraform is out of date! The latest version
       is 1.0.3. You can update by downloading from https://www.terraform.io/downloads.html
$$$$$$ Finished reading the Terraform client version.
$$$$$$ Finished verifying the Terraform client version.
$$$$$$ Initializing the Terraform working directory...
       Initializing modules...
       
       Initializing the backend...
       
       Initializing provider plugins...
       - Using previously-installed hashicorp/random v3.1.0
       - Using previously-installed hashicorp/google v3.78.0
       - Using previously-installed hashicorp/null v3.1.0
       - Using previously-installed hashicorp/google-beta v3.78.0
       
       The following providers do not have any version constraints in configuration,
       so the latest version was installed.
       
       To prevent automatic upgrades to new major versions that may contain breaking
       changes, we recommend adding version constraints in a required_providers block
       in your configuration, with the constraint strings suggested below.
       
       * hashicorp/google-beta: version = "~> 3.78.0"
       
       Terraform has been successfully initialized!
$$$$$$ Finished initializing the Terraform working directory.
$$$$$$ Selecting the kitchen-terraform-postgresql-public-iam-local Terraform workspace...
       
       Workspace "kitchen-terraform-postgresql-public-iam-local" doesn't exist.
       
       You can create this workspace with the "new" subcommand.
$$$$$$ Creating the kitchen-terraform-postgresql-public-iam-local Terraform workspace...
       Created and switched to workspace "kitchen-terraform-postgresql-public-iam-local"!
       
       You're now on a new, empty workspace. Workspaces isolate their state,
       so if you run "terraform plan" Terraform will not see any existing state
       for this configuration.
$$$$$$ Finished creating the kitchen-terraform-postgresql-public-iam-local Terraform workspace.
$$$$$$ Destroying the Terraform-managed infrastructure...
       
       Error: No value for required variable
       
         on variables.tf line 36:
         36: variable "cloudsql_pg_sa" {
       
       The root module input variable "cloudsql_pg_sa" is not set, and has no default
       value. Use a -var or -var-file command line argument to provide a value for
       this variable.
       
>>>>>> Running the command `terraform destroy -auto-approve -lock=true -lock-timeout=0s -input=false  -parallelism=10 -refresh=true  ` failed due to a non-zero exit code of 1.
-----> Destroying <mysql-private-local>...
$$$$$$ Verifying the Terraform client version is in the supported interval of >= 0.11.4, < 0.14.0...
$$$$$$ Reading the Terraform client version...
       Terraform v0.13.5
       + provider registry.terraform.io/hashicorp/google v3.78.0
       + provider registry.terraform.io/hashicorp/google-beta v3.78.0
       + provider registry.terraform.io/hashicorp/null v3.1.0
       + provider registry.terraform.io/hashicorp/random v3.1.0
       
       Your version of Terraform is out of date! The latest version
       is 1.0.3. You can update by downloading from https://www.terraform.io/downloads.html
$$$$$$ Finished reading the Terraform client version.
$$$$$$ Finished verifying the Terraform client version.
$$$$$$ Initializing the Terraform working directory...
       Initializing modules...
       
       Initializing the backend...
       
       Initializing provider plugins...
       - Using previously-installed hashicorp/google-beta v3.78.0
       - Using previously-installed hashicorp/random v3.1.0
       - Using previously-installed hashicorp/google v3.78.0
       - Using previously-installed hashicorp/null v3.1.0
       
       Terraform has been successfully initialized!
$$$$$$ Finished initializing the Terraform working directory.
$$$$$$ Selecting the kitchen-terraform-mysql-private-local Terraform workspace...
       
       Workspace "kitchen-terraform-mysql-private-local" doesn't exist.
       
       You can create this workspace with the "new" subcommand.
$$$$$$ Creating the kitchen-terraform-mysql-private-local Terraform workspace...
       Created and switched to workspace "kitchen-terraform-mysql-private-local"!
       
       You're now on a new, empty workspace. Workspaces isolate their state,
       so if you run "terraform plan" Terraform will not see any existing state
       for this configuration.
$$$$$$ Finished creating the kitchen-terraform-mysql-private-local Terraform workspace.
$$$$$$ Destroying the Terraform-managed infrastructure...
       
       Destroy complete! Resources: 0 destroyed.
$$$$$$ Finished destroying the Terraform-managed infrastructure.
$$$$$$ Selecting the default Terraform workspace...
       Switched to workspace "default".
$$$$$$ Finished selecting the default Terraform workspace.
$$$$$$ Deleting the kitchen-terraform-mysql-private-local Terraform workspace...
       Deleted workspace "kitchen-terraform-mysql-private-local"!
$$$$$$ Finished deleting the kitchen-terraform-mysql-private-local Terraform workspace.
       Finished destroying <mysql-private-local> (0m7.87s).
-----> Destroying <mssql-public-local>...
$$$$$$ Verifying the Terraform client version is in the supported interval of >= 0.11.4, < 0.14.0...
$$$$$$ Reading the Terraform client version...
       Terraform v0.13.5
       + provider registry.terraform.io/hashicorp/google v3.78.0
       + provider registry.terraform.io/hashicorp/google-beta v3.78.0
       + provider registry.terraform.io/hashicorp/null v3.1.0
       + provider registry.terraform.io/hashicorp/random v3.1.0
       
       Your version of Terraform is out of date! The latest version
       is 1.0.3. You can update by downloading from https://www.terraform.io/downloads.html
$$$$$$ Finished reading the Terraform client version.
$$$$$$ Finished verifying the Terraform client version.
$$$$$$ Initializing the Terraform working directory...
       Initializing modules...
       
       Initializing the backend...
       
       Initializing provider plugins...
       - Using previously-installed hashicorp/random v3.1.0
       - Using previously-installed hashicorp/google-beta v3.78.0
       - Using previously-installed hashicorp/google v3.78.0
       - Using previously-installed hashicorp/null v3.1.0
       
       The following providers do not have any version constraints in configuration,
       so the latest version was installed.
       
       To prevent automatic upgrades to new major versions that may contain breaking
       changes, we recommend adding version constraints in a required_providers block
       in your configuration, with the constraint strings suggested below.
       
       * hashicorp/google: version = "~> 3.78.0"
       * hashicorp/null: version = "~> 3.1.0"
       * hashicorp/random: version = "~> 3.1.0"
       
       Terraform has been successfully initialized!
$$$$$$ Finished initializing the Terraform working directory.
$$$$$$ Selecting the kitchen-terraform-mssql-public-local Terraform workspace...
       
       Workspace "kitchen-terraform-mssql-public-local" doesn't exist.
       
       You can create this workspace with the "new" subcommand.
$$$$$$ Creating the kitchen-terraform-mssql-public-local Terraform workspace...
       Created and switched to workspace "kitchen-terraform-mssql-public-local"!
       
       You're now on a new, empty workspace. Workspaces isolate their state,
       so if you run "terraform plan" Terraform will not see any existing state
       for this configuration.
$$$$$$ Finished creating the kitchen-terraform-mssql-public-local Terraform workspace.
$$$$$$ Destroying the Terraform-managed infrastructure...
       
       Destroy complete! Resources: 0 destroyed.
$$$$$$ Finished destroying the Terraform-managed infrastructure.
$$$$$$ Selecting the default Terraform workspace...
       Switched to workspace "default".
$$$$$$ Finished selecting the default Terraform workspace.
$$$$$$ Deleting the kitchen-terraform-mssql-public-local Terraform workspace...
       Deleted workspace "kitchen-terraform-mssql-public-local"!
$$$$$$ Finished deleting the kitchen-terraform-mssql-public-local Terraform workspace.
       Finished destroying <mssql-public-local> (0m7.23s).
-----> Destroying <mysql-ha-local>...
$$$$$$ Verifying the Terraform client version is in the supported interval of >= 0.11.4, < 0.14.0...
$$$$$$ Reading the Terraform client version...
       Terraform v0.13.5
       + provider registry.terraform.io/hashicorp/google v3.78.0
       + provider registry.terraform.io/hashicorp/google-beta v3.78.0
       + provider registry.terraform.io/hashicorp/null v3.1.0
       + provider registry.terraform.io/hashicorp/random v3.1.0
       
       Your version of Terraform is out of date! The latest version
       is 1.0.3. You can update by downloading from https://www.terraform.io/downloads.html
$$$$$$ Finished reading the Terraform client version.
$$$$$$ Finished verifying the Terraform client version.
$$$$$$ Initializing the Terraform working directory...
       Initializing modules...
       
       Initializing the backend...
       
       Initializing provider plugins...
       - Using previously-installed hashicorp/google v3.78.0
       - Using previously-installed hashicorp/null v3.1.0
       - Using previously-installed hashicorp/random v3.1.0
       - Using previously-installed hashicorp/google-beta v3.78.0
       
       The following providers do not have any version constraints in configuration,
       so the latest version was installed.
       
       To prevent automatic upgrades to new major versions that may contain breaking
       changes, we recommend adding version constraints in a required_providers block
       in your configuration, with the constraint strings suggested below.
       
       * hashicorp/google-beta: version = "~> 3.78.0"
       
       Terraform has been successfully initialized!
$$$$$$ Finished initializing the Terraform working directory.
$$$$$$ Selecting the kitchen-terraform-mysql-ha-local Terraform workspace...
       
       Workspace "kitchen-terraform-mysql-ha-local" doesn't exist.
       
       You can create this workspace with the "new" subcommand.
$$$$$$ Creating the kitchen-terraform-mysql-ha-local Terraform workspace...
       Created and switched to workspace "kitchen-terraform-mysql-ha-local"!
       
       You're now on a new, empty workspace. Workspaces isolate their state,
       so if you run "terraform plan" Terraform will not see any existing state
       for this configuration.
$$$$$$ Finished creating the kitchen-terraform-mysql-ha-local Terraform workspace.
$$$$$$ Destroying the Terraform-managed infrastructure...
       
       Destroy complete! Resources: 0 destroyed.
$$$$$$ Finished destroying the Terraform-managed infrastructure.
$$$$$$ Selecting the default Terraform workspace...
       Switched to workspace "default".
$$$$$$ Finished selecting the default Terraform workspace.
$$$$$$ Deleting the kitchen-terraform-mysql-ha-local Terraform workspace...
       Deleted workspace "kitchen-terraform-mysql-ha-local"!
$$$$$$ Finished deleting the kitchen-terraform-mysql-ha-local Terraform workspace.
       Finished destroying <mysql-ha-local> (0m7.17s).
-----> Destroying <postgresql-ha-local>...
$$$$$$ Verifying the Terraform client version is in the supported interval of >= 0.11.4, < 0.14.0...
$$$$$$ Reading the Terraform client version...
       Terraform v0.13.5
       + provider registry.terraform.io/hashicorp/google v3.78.0
       + provider registry.terraform.io/hashicorp/google-beta v3.78.0
       + provider registry.terraform.io/hashicorp/null v3.1.0
       + provider registry.terraform.io/hashicorp/random v3.1.0
       
       Your version of Terraform is out of date! The latest version
       is 1.0.3. You can update by downloading from https://www.terraform.io/downloads.html
$$$$$$ Finished reading the Terraform client version.
$$$$$$ Finished verifying the Terraform client version.
$$$$$$ Initializing the Terraform working directory...
       Initializing modules...
       
       Initializing the backend...
       
       Initializing provider plugins...
       - Using previously-installed hashicorp/null v3.1.0
       - Using previously-installed hashicorp/google-beta v3.78.0
       - Using previously-installed hashicorp/random v3.1.0
       - Using previously-installed hashicorp/google v3.78.0
       
       The following providers do not have any version constraints in configuration,
       so the latest version was installed.
       
       To prevent automatic upgrades to new major versions that may contain breaking
       changes, we recommend adding version constraints in a required_providers block
       in your configuration, with the constraint strings suggested below.
       
       * hashicorp/google-beta: version = "~> 3.78.0"
       
       Terraform has been successfully initialized!
$$$$$$ Finished initializing the Terraform working directory.
$$$$$$ Selecting the kitchen-terraform-postgresql-ha-local Terraform workspace...
       
       Workspace "kitchen-terraform-postgresql-ha-local" doesn't exist.
       
       You can create this workspace with the "new" subcommand.
$$$$$$ Creating the kitchen-terraform-postgresql-ha-local Terraform workspace...
       Created and switched to workspace "kitchen-terraform-postgresql-ha-local"!
       
       You're now on a new, empty workspace. Workspaces isolate their state,
       so if you run "terraform plan" Terraform will not see any existing state
       for this configuration.
$$$$$$ Finished creating the kitchen-terraform-postgresql-ha-local Terraform workspace.
$$$$$$ Destroying the Terraform-managed infrastructure...
       
       Destroy complete! Resources: 0 destroyed.
$$$$$$ Finished destroying the Terraform-managed infrastructure.
$$$$$$ Selecting the default Terraform workspace...
       Switched to workspace "default".
$$$$$$ Finished selecting the default Terraform workspace.
$$$$$$ Deleting the kitchen-terraform-postgresql-ha-local Terraform workspace...
       Deleted workspace "kitchen-terraform-postgresql-ha-local"!
$$$$$$ Finished deleting the kitchen-terraform-postgresql-ha-local Terraform workspace.
       Finished destroying <postgresql-ha-local> (0m7.08s).
-----> Destroying <mssql-ha-local>...
$$$$$$ Verifying the Terraform client version is in the supported interval of >= 0.11.4, < 0.14.0...
$$$$$$ Reading the Terraform client version...
       Terraform v0.13.5
       + provider registry.terraform.io/hashicorp/google v3.78.0
       + provider registry.terraform.io/hashicorp/google-beta v3.78.0
       + provider registry.terraform.io/hashicorp/null v3.1.0
       + provider registry.terraform.io/hashicorp/random v3.1.0
       
       Your version of Terraform is out of date! The latest version
       is 1.0.3. You can update by downloading from https://www.terraform.io/downloads.html
$$$$$$ Finished reading the Terraform client version.
$$$$$$ Finished verifying the Terraform client version.
$$$$$$ Initializing the Terraform working directory...
       Initializing modules...
       
       Initializing the backend...
       
       Initializing provider plugins...
       - Using previously-installed hashicorp/random v3.1.0
       - Using previously-installed hashicorp/google v3.78.0
       - Using previously-installed hashicorp/google-beta v3.78.0
       - Using previously-installed hashicorp/null v3.1.0
       
       The following providers do not have any version constraints in configuration,
       so the latest version was installed.
       
       To prevent automatic upgrades to new major versions that may contain breaking
       changes, we recommend adding version constraints in a required_providers block
       in your configuration, with the constraint strings suggested below.
       
       * hashicorp/google: version = "~> 3.78.0"
       * hashicorp/null: version = "~> 3.1.0"
       * hashicorp/random: version = "~> 3.1.0"
       
       Terraform has been successfully initialized!
$$$$$$ Finished initializing the Terraform working directory.
$$$$$$ Selecting the kitchen-terraform-mssql-ha-local Terraform workspace...
       
       Workspace "kitchen-terraform-mssql-ha-local" doesn't exist.
       
       You can create this workspace with the "new" subcommand.
$$$$$$ Creating the kitchen-terraform-mssql-ha-local Terraform workspace...
       Created and switched to workspace "kitchen-terraform-mssql-ha-local"!
       
       You're now on a new, empty workspace. Workspaces isolate their state,
       so if you run "terraform plan" Terraform will not see any existing state
       for this configuration.
$$$$$$ Finished creating the kitchen-terraform-mssql-ha-local Terraform workspace.
$$$$$$ Destroying the Terraform-managed infrastructure...
       
       Destroy complete! Resources: 0 destroyed.
$$$$$$ Finished destroying the Terraform-managed infrastructure.
$$$$$$ Selecting the default Terraform workspace...
       Switched to workspace "default".
$$$$$$ Finished selecting the default Terraform workspace.
$$$$$$ Deleting the kitchen-terraform-mssql-ha-local Terraform workspace...
       Deleted workspace "kitchen-terraform-mssql-ha-local"!
$$$$$$ Finished deleting the kitchen-terraform-mssql-ha-local Terraform workspace.
       Finished destroying <mssql-ha-local> (0m7.09s).
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Destroy failed on instance <postgresql-public-iam-local>.  Please see .kitchen/logs/postgresql-public-iam-local.log for more details
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
```

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by eahrend** [comment]
Is this not needed anymore?

File: modules/mysql/variables.tf [hunk 1]
```diff
@@ -246,6 +246,7 @@ variable "additional_users" {
     name     = string
     password = string
     host     = string
+    type     = string
```
Comment (morgante) [inline modules/mysql/variables.tf:249]: Since adding a field to an object breaks backwards compatibility, could you convert this to a `list(map(any))`?

**Review by morgante — CHANGES_REQUESTED** [review]
Thanks for the PR and apologies that I missed this earlier.

File: modules/mysql/variables.tf (hunk 1 above)
Comment (eahrend) [reply modules/mysql/variables.tf:249]: Sure. I'll have it swapped first thing tomorrow morning

**Comment by eahrend** [comment]
@morgante changed additional users to list(map(any)), fixed some typos and a test. I am still getting an error when running the `make docker_test_integration` command, but it doesn't look like it's actually causing any issues still. 
output:
https://gist.github.com/eahrend/80adb6026a4aafc7a63b0061c0ffe96a

**Comment by morgante** [comment]
@eahrend It looks like integration tests are passing, but the linters are [now failing](https://github.com/terraform-google-modules/terraform-google-sql-db/pull/237#issuecomment-891964146).

**Comment by eahrend** [comment]
Ah, apologies. I'll have this tenant asap tomorrow morning.

**Comment by eahrend** [comment]
Alright, updated it. 

There does seem to be something weird happening locally, if I have `.terraform` files existing before running `make docker_test_lint`, like from running integration tests or having a failed `make docker_test_lint` will cause the check_headers test to fail. Removing the existing `.terraform` files fixes this, but it may make more sense to create a separate ticket to just ignore `.terraform` files during the check_header test. 


```
Checking file headers
./examples/postgresql-public-iam/.terraform.lock.hcl
./examples/mysql-public/.terraform.lock.hcl
./examples/mysql-private/.terraform.lock.hcl
./modules/mysql/.terraform.lock.hcl
./examples/mssql-public/.terraform.lock.hcl
./examples/mysql-ha/.terraform.lock.hcl
./examples/postgresql-ha/.terraform.lock.hcl
./examples/postgresql-public/.terraform.lock.hcl
./test/fixtures/postgresql-ha/.terraform.lock.hcl
./modules/postgresql/.terraform.lock.hcl
./modules/mssql/.terraform.lock.hcl
./test/fixtures/postgresql-public/.terraform.lock.hcl
./test/fixtures/mssql-public/.terraform.lock.hcl
./test/fixtures/mysql-ha/.terraform.lock.hcl
./test/fixtures/mysql-private/.terraform.lock.hcl
./test/setup/.terraform.lock.hcl
./modules/private_service_access/.terraform.lock.hcl
./test/fixtures/mssql-ha/.terraform.lock.hcl
./test/fixtures/mysql-public/.terraform.lock.hcl
./modules/safer_mysql/.terraform.lock.hcl
./test/fixtures/postgresql-public-iam/.terraform.lock.hcl
```

**Comment by morgante** [comment]
> Alright, updated it.
> 
> There does seem to be something weird happening locally, if I have `.terraform` files existing before running `make docker_test_lint`, like from running integration tests or having a failed `make docker_test_lint` will cause the check_headers test to fail. Removing the existing `.terraform` files fixes this, but it may make more sense to create a separate ticket to just ignore `.terraform` files during the check_header test.
> 
> ```
> Checking file headers
> ./examples/postgresql-public-iam/.terraform.lock.hcl
> ./examples/mysql-public/.terraform.lock.hcl
> ./examples/mysql-private/.terraform.lock.hcl
> ./modules/mysql/.terraform.lock.hcl
> ./examples/mssql-public/.terraform.lock.hcl
> ./examples/mysql-ha/.terraform.lock.hcl
> ./examples/postgresql-ha/.terraform.lock.hcl
> ./examples/postgresql-public/.terraform.lock.hcl
> ./test/fixtures/postgresql-ha/.terraform.lock.hcl
> ./modules/postgresql/.terraform.lock.hcl
> ./modules/mssql/.terraform.lock.hcl
> ./test/fixtures/postgresql-public/.terraform.lock.hcl
> ./test/fixtures/mssql-public/.terraform.lock.hcl
> ./test/fixtures/mysql-ha/.terraform.lock.hcl
> ./test/fixtures/mysql-private/.terraform.lock.hcl
> ./test/setup/.terraform.lock.hcl
> ./modules/private_service_access/.terraform.lock.hcl
> ./test/fixtures/mssql-ha/.terraform.lock.hcl
> ./test/fixtures/mysql-public/.terraform.lock.hcl
> ./modules/safer_mysql/.terraform.lock.hcl
> ./test/fixtures/postgresql-public-iam/.terraform.lock.hcl
> ```

I think some of our future test cleanups will help with this. Thanks for your patience!

**Review by morgante — APPROVED** [review]

**Pull request merged**